Mathlib/MeasureTheory/Integral/Lebesgue.lean

```diff
@@ -1128,7 +1128,7 @@ theorem lintegral_liminf_le {f : ℕ → α → ℝ≥0∞} (h_meas : ∀ n, Mea
   lintegral_liminf_le' fun n => (h_meas n).aemeasurable
 #align measure_theory.lintegral_liminf_le MeasureTheory.lintegral_liminf_le
 
-theorem limsup_lintegral_le {f : ℕ → α → ℝ≥0∞} {g : α → ℝ≥0∞} (hf_meas : ∀ n, Measurable (f n))
+theorem limsup_lintegral_le {f : ℕ → α → ℝ≥0∞} (g : α → ℝ≥0∞) (hf_meas : ∀ n, Measurable (f n))
```

Comment:
AFAIR, you've changed from explicit to implicit in some other cases. Does `(g := _)` work for you here?

Reply:
This is different from the other cases you are thinking about in that `g` appears neither on the LHS or RHS, so it can only be unified for if we already have proofs of `hf_meas` and `h_bound`, which is impractical.